Front-End/challenge2/script.js

```diff
@@ -5,3 +5,97 @@ TO-DO:
 - The calculator should be completely functional
 
 */
+const zero = document.getElementById("zero");
+const one = document.getElementById("one");
+const two = document.getElementById("two");
+const three = document.getElementById("three");
+const four = document.getElementById("four");
+const five = document.getElementById("five");
+const six = document.getElementById("six");
+const seven = document.getElementById("seven");
+const eight = document.getElementById("eight");
+const nine = document.getElementById("nine");
+const display = document.getElementById("display");
+const add = document.getElementById('add')
+const equals = document.getElementById('equals')
+const subtrack = document.getElementById('subtrack')
+const multiplication = document.getElementById('multiplication')
+const division = document.getElementById('division')
+let touch = 0;
+let result = 0;
+let flag = false;
+
+function evaluate(fn) {
+  return new Function('return ' + fn)();
+}
+
+function addValue(value) {
+  if (touch == 0) {
+    display.innerHTML = value;
+    touch = 1
+  } else {
+    if (value == "0" && display.textContent == "0") {
+      return
+    } else {
+      if (display.textContent == "0") {
+        display.innerHTML = value
+      } else {
+        display.innerHTML += value;
+      }
+
+    }
+  }
+
+}
+
+function addSymbol(value) {
+  if (touch == 0 && flag == false) {
+    return
+  } else {
+    touch = 1
+    if (display.textContent[display.textContent.length - 1] == value || !parseInt(display.textContent[display.textContent.length - 1])) {
```

Comment:
please use variables instead of putting a long evaluation in the `if` 
eg:
const result = your code here;

if(result)

Reply:
I'll do that from now on, thanks